pkg/sql/codegen/xgboost/docgen.go

```diff
@@ -36,8 +36,9 @@ const docTemplateText = `# XGBoost Parameters
 SELECT * FROM boston.train
 TRAIN xgboost.gbtree
 WITH
-    objective="reg:squarederror",
-    train.num_boost_round = 30
+    objective ="reg:squarederror",
+    train.num_boost_round = 30,
+    validation.select = "SELECT * FROM boston.train LIMIT 8"
```

Comment:
The validation data is usually not the same as the training data. So maybe rename `boston.train` to `boston.val`?

Reply:
sure